GeoLibrary.Unit/GeoLibrary.Unit.csproj

```diff
@@ -1,16 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
The idea to use a little early version of .netcore is that the project which uses this lib doesn't have to use a new .net version if I understand the depencencies correctly.

If yes, I would suggest to keep the version (or change it to a LTS version but still earlier one than the latest.

If not, I'm OK with the latest .net version.

Reply:
The v6.0 is an LTS, support for v3.1 ends this december --> https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core.

Moreover this is only the Unit-Testing project. The main lib still references `netstandard2.0`.
Are you ok with this? :)

Reply:
I didn't realize this is just the unit test project. Then it should be OK